eftest/src/eftest/runner.clj

```diff
@@ -8,6 +8,9 @@
             [eftest.output-capture :as capture])
   (:import [java.util.concurrent Executors ExecutorService]))
 
+(defprotocol Executor
```

Comment:
I don't understand the reason for adding this protocol. It seems to use more code and indirection to achieve the same result as before.